enterprise-suite/tests/smokecommon

```diff
@@ -62,10 +73,15 @@ test_summary() {
     exit $TEST_FAILURES
 }
 
+
 # Get URL for NodePort assignment
 # nodeport <name>
 nodeport() {
-  minikube service --namespace=lightbend --url $1
+  if [[ $context == "minikube" ]]; then
+    minikube service --namespace=lightbend --url $1
+  elif [[ $context == "GKE" ]]; then
+    get_ing_ip $1-ing
```

Comment:
else ....  case please, we should error out if in an unsupported context

Reply:
since this is a global and about kubernetes, let's call this `$KubetnetesContext` instead of `$context`